pkgs/development/python-modules/cfv/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "cfv";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "cfv-project";
+    repo = "cfv";
+    rev = "v${version}";
```

Comment:
`refs/tags/v${version}` would be safer in case there is ever a branch matching the tag name

Reply:
Like this? ad8217c

Reply:
thanks, that is good